### ISSUE TITLE
[Suggestion] DME chams for dead players and weapon

### ISSUE THREAD

**Issue by suddenlybydl0** [danielkrupinski/Osiris#1187]
Chams for dead players and weapon in DME hook

**Comment by effex1337** [comment]
Ragdolls and weapons don't go through dme.

**Comment by suddenlybydl0** [comment]
> Ragdolls and weapons don't go through dme.
Yes, but there is a way to get ragdoll and weapons to go through DME.
I would do it myself, but I'm a dumb paster, so it’s not in my power.
https://www.unknowncheats.me/forum/counterstrike-global-offensive/366471-dme-weapon-chams.html

**Comment by Sapphire1337420** [comment]
@danielkrupinski
